user_docs/en/changes.t2t

```diff
@@ -8,9 +8,9 @@
 == New Features ==
 - A Braille setting has been added to "show messages indefinitely". (#6669)
 - In Microsoft Outlook message lists, report if a message is flagged. (#6374)
-- In Microsoft Powerpoint, the exact type of a shape is now reported when editing a slide (Examples include: triangle, circle, video, arrow), rather than just 'shape'. (#7111)
+- In Microsoft PowerPoint, the exact type of a shape is now reported when editing a slide (Examples include: triangle, circle, video, arrow), rather than just 'shape'. (#7111)

```

Comment:
This "examples include" thing is a bit weird linguistically. Maybe we could change it to this:

> `- In Microsoft PowerPoint, the exact type of a shape is now reported when editing a slide (such as triangle, circle, video or arrow), rather than just "shape". (#7111)`